great_expectations/core/factory/suite_factory.py

```diff
@@ -125,3 +125,23 @@ def all(self) -> Iterable[ExpectationSuite]:
                 self._store.submit_all_deserialization_event(e)
                 raise
         return deserializable_suites
+
+    @public_api
+    def add_or_update(self, suite: ExpectationSuite) -> ExpectationSuite:
```

Comment:
Could we add a docstring here please?

Reply:
thanks for catching! meant to do that and missed it. added in [5acc32a](https://github.com/great-expectations/great_expectations/pull/10796/commits/5acc32a3245ac75792c5d316fbd2a434843f05b7)